Makefile

```diff
@@ -24,6 +24,8 @@ BUILD_PARAMETER = $(LIGO) compile-parameter --syntax cameligo
 escape_quote = $(subst ','\'',$(1))
 # Utility function to escape double quotes
 escape_double_quote = $(subst $\",$\\",$(1))
+# Utility function to require a variable to be defined
+require_defined = $(if $(value $1),$(value $1),$(error "$1 needs to be defined"))
```

Comment:
Does this work if `$1` is zero?

Reply:
It seems to work fine if `$1` is `0`. It only throws error if the value `$1` is an empty string.